README.md

```diff
@@ -70,7 +70,7 @@ func main() {
 	// Get the last 4 messages
 	from := uint32(1)
 	to := mbox.Messages
-	if mbox.Messages > 3 {
+	if to > 3 {
```

Comment:
This condition is checking whether `mbox.Messages - 3 > 0` (see the comment). I think it makes more sense to keep it as-is.

Reply:
yes, the if the condition it does make a lot of sense. My suggestion is not regarding the implementation but the usage of a variable that it was assigned just right before the if:


to has already mbox.Messages assigned that's why I thought it would make sense to use it in the if condition.

Reply:
The variable is used after, in the `AddRange` call.